python_examples/age_stratified_example/age_hist/age_hist.py

```diff
@@ -0,0 +1,70 @@
+import pyEpiabm as pe
+import matplotlib.pyplot as plt
+import numpy as np
+import os
+
+# Set config file for Parameters
+pe.Parameters.set_file(os.path.join(os.path.dirname(__file__), os.pardir,
+                       "simple_parameters_with_age.json"))
+
+# Set population parameters
+pop_params = {"population_size": 1000000, "cell_number": 1,
+              "microcell_number": 1, "household_number": 5,
+              "place_number": 2}
+
+# Create a population based on the parameters given.
+population = pe.routine.ToyPopulationFactory().make_pop(pop_params)
+
+xlist = []
+# Create array of density of population by age
+for cell in population.cells:
+    for person in cell.persons:
+        xlist.append(person.age)
+
+x = np.array(xlist)
+
+# Plot and save histogram
+plt.figure(1)
+plt.hist(x, density=False, rwidth=0.6, bins=17)
+plt.ylabel('Count')
+plt.xlabel('Age')
+plt.title('Histogram of toy population')
+plt.savefig('python_examples/age_stratified_example/age_hist/Withoutsim.png')
```

Comment:
Is it possible to group this into a function, just to avoid duplication of code? Not a big deal but just looks a little ugly!

Reply:
It's a bit ugly but it works. You'd have to have some of the lines outside the function anyways